custom-plugins/grafana-region-map-panel/package.json

```
@@ -4,7 +4,8 @@
   "description": "Region Map Panel Plugin for Grafana",
   "main": "src/module.js",
   "scripts": {
-    "build": "node --max-old-space-size=8192 ./node_modules/.bin/webpack --config webpack.config.prod.js",
+    "copy": "rsync -rP --exclude=node_modules ./ ~/GitHub/msupply-dashboard/custom-plugins/grafana-region-map-panel/",
```

```suggestion
    "copy": "rsync -rP --exclude=node_modules ./ ~/GitHub/msupply-dashboard/custom-plugins/grafana-region-map-panel/",
    "get": "rsync -rP --exclude=node_modules ~/GitHub/msupply-dashboard/custom-plugins/grafana-region-map-panel/ ./",
```

You could make it two-way with this? I've tested it and it seems to work, but any risks you can think of?

---

great idea! I'd missed that, but it's an important point - to ensure that the dev folder is up to date before you start.
Can't see a downside if this isn't part of the build command, surely it's up to the dev to not screw it up then :)